community/learn/graphql-tutorials/tutorials/react-apollo/tutorial-site/content/queries/2-create-query.md

```diff
@@ -142,9 +142,9 @@ class TodoPrivateList extends Component {
 +
 +    let filteredTodos = todos;
     if (this.state.filter === "active") {
-      filteredTodos = this.state.todos.filter(todo => todo.is_completed !== true);
+      filteredTodos = todos.filter(todo => todo.is_completed !== true);
```

Comment:
Include the old code with a `-` to indicate removing the old line and including the new line.